src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -102,19 +106,22 @@ private Future<Void> checkTransactions(Budget budget, DBClient client) {
 
     client.getPgClient().execute(sql, reply -> {
         if (reply.failed()) {
-          logger.error("Transaction retrieval by query {} failed", sql, reply.cause());
+          logger.error("checkTransactions:: Transaction retrieval by query {} failed", sql, reply.cause());
```

Comment:
The same, better to include budget.getFundId()

Reply:
Added